src/bisect.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+/* @flow */
+
+var sign = require('./sign');
+/**
+ * [Bisection method](https://en.wikipedia.org/wiki/Bisection_method) is a root-finding 
+ * method that repeatedly bisects an interval to find the root.
+ * 
+ * This function returns a numerical approximation to the exact value.
+ * 
+ * @param {Function} f input function
+ * @param {Number} a start of interval
```

Comment:
the argument names in JavaScript are updated 👍  but the `@param` tags will also need to be updated